pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/golang/protobuf/ptypes/timestamp"
```

Comment:
I don't believe we can take a dependency on  the protobuf `Timestamp` in the CRDs. We should probably use [`metav1.Time`] which is the K8s native API type for representing timestamps.

Reply:
I might be able to use int64 similar to how its been used by `LastHeartbeatTime` in the `AzDriverNode` object

Reply:
I'd rather use the standard types for timestamps. `LastHeartbeatTime` should probably be a `metav1.Time` as well.